build-support/bin/native/calculate_engine_hash.sh

```diff
@@ -36,9 +36,7 @@ function calculate_current_hash() {
     git ls-files --cached --others --exclude-standard \
      "${NATIVE_ROOT}" \
      "${REPO_ROOT}/rust-toolchain" \
-     "${REPO_ROOT}/src/python/pants/engine/internals/native.py" \
```

Comment:
These were left behind from the CFFI days: `native.py` is no longer imported during the process of building the engine.